### PR TITLE
 Consistent calculations with multipliers, and consistently round scores 

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

File: iatidataquality/organisations.py [hunk 1]
```diff
@@ -285,7 +285,7 @@ def agg_detail(agt):
     agg_type = [agg_detail(agt) for agt in all_aggregation_types]
 
     timeliness_score = get_timeliness_score(organisation.frequency, organisation.timelag)
-    max_points = round(33.3 + 66.7 * timeliness_score, 1)
+    max_points = round(33.3 + 66.7 * timeliness_score, 2)
```
Comment (simon-20) [inline iatidataquality/organisations.py:288]: @Bjwebb, should the `33.3` here be `33.33`, and the `66.7` be `66.67`? Sections 3.3 and 3.4 of the technical paper use the figures `33.33`and `66.67` in the worked examples.

File: unittests/test_web.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ def test_organisation_publication_page(
     assert response.status_code == 200
 
     assert (
-        '<p class="lead">It looks like you publish quarterly with a time lag of more than a quarter, so the maximum you can score for IATI data is 85.0 points. The total points for the relevant indicators have been adjusted accordingly.'
+        '<p class="lead">It looks like you publish quarterly with a time lag of more than a quarter, so the maximum you can score for IATI data is 84.99 points. The total points for the relevant indicators have been adjusted accordingly.'
```
Comment (simon-20) [inline unittests/test_web.py:99]: And if the `33.3` should be `33.33`, and same for `66.7` figure, then the max score should be `85.00`, I think, even when rounding to do digits:

```
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = 100 / 1.5
>>> a
66.66666666666667
>>> b = a * 0.775
>>> b
51.66666666666667
>>> c = b + 33.33
>>> c
84.99666666666667
>>> round(c, 2)
85.0
>>> 
```

Unless the weighting figures have been updated?

Comment (Bjwebb) [reply unittests/test_web.py]: Yes, that's right, I've updated it to fix it.

File: iatidataquality/organisations.py (hunk 1 above)
Comment (Bjwebb) [reply iatidataquality/organisations.py]: I've updated it to use 100.0/3, which is what we use for the actual calculation.

**Review by simon-20 — APPROVED** [review]

**Pull request merged**